src/main/resources/application.properties

```diff
@@ -1,8 +1,11 @@
 spring.application.name=spring-ecommerce
-spring.datasource.url=jdbc:h2:file:./data/productDb
+spring.datasource.url=jdbc:h2:mem:database
 spring.datasource.driver-class-name=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=
 spring.h2.console.enabled=true
 spring.sql.init.mode=always
-spring.sql.init.schema-locations=classpath:schema.sql
\ No newline at end of file
+spring.sql.init.schema-locations=classpath:schema.sql
+spring.sql.init.data-locations=classpath:data.sql
+security.jwt.token.secret-key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9.ih1aovtQShabQ7l0cINw4k1fagApg3qLWiB8Kt59Lno
```

Comment:
Please consider ways to manage sensitive information without uploading it to GitHub 🙂